src/utils.coffee

```diff
@@ -36,6 +36,16 @@ exports.mixpanelTrack = (event, properties = {}) ->
 			message: properties.error.message
 			stack: properties.error.stack
 
+	# Don't log private env vars (e.g. api keys)
+	if properties?.app?.env?
+		try
+			{ env } = properties.app
+			env = JSON.parse(env) if typeof env is 'string'
+			safeEnv = _.omit(env, config.privateAppEnvVars)
+			properties.app.env = JSON.stringify(safeEnv)
+		catch
+			_.noop
```

Comment:
Although maybe if we hit an error checking for env vars to hide we should just set `properties.app.env = 'Fully hidden due to error in selective hiding'` or something like that